### ISSUE TITLE
Accessibility issue with currently active button in Resources

### ISSUE THREAD

**Issue by robmausser** [adaptlearning/adapt-contrib-resources#72]
Screen reader does not announce the selected state of currently active button. Screen reader should announce the selected state of currently active button. Without the selected state, screen reader users would not be able to identify the currently active link. Visit this link for the code:
https://www.w3.org/WAI/PF/aria/states_and_properties#aria-selected

https://www.screencast.com/t/1b2xs9Gu9

**Comment by oliverfoster** [comment]
Duplicate of https://github.com/adaptlearning/adapt-contrib-resources/issues/71
